get-started/relays.mdx

```diff
@@ -3,11 +3,88 @@ title: Relay Guide
\ No newline at end of file
+    ```javascript
+    // Import the ethers library
+    const ethers = require("ethers");
+
+    // Define the provider using the RPC URL
+    const provider = new ethers.JsonRpcProvider("https://chainrpc.testnet.mev-commit.xyz/");
+
+    // Define the contract address and ABI
+    const contractAddress = "0x5960774AD41D03DAB4916a30bD2190f8b3b3b4b2";
+    const abi = [
+        "event ProviderRegistered(address indexed provider, uint256 stakedAmount)"
+    ];
+
+    // Create a new contract instance
+    const contract = new ethers.Contract(contractAddress, abi, provider);
+
+    // Function to retrieve the list of registered providers
+    async function getRegisteredProviders() {
+        // Create a filter for the ProviderRegistered event
+        const filter = contract.filters.ProviderRegistered();
+
+        // Query the contract for the ProviderRegistered events
+        const events = await contract.queryFilter(filter);
+
+        // Map the events to get the list of provider addresses
+        const providers = events.map(event => event.args.provider);
+
+        return providers;
+    }
+
+    // Call the function and log the result
+    getRegisteredProviders().then(providers => {
+        console.log("Registered Providers:", providers);
+    }).catch(error => {
+        console.error("Error:", error);
+    });
+    ```
+
+  </Accordion>
+  
+  <Accordion title="Expected Output">
+    The output will be a list of addresses that have registered with mev-commit.
+
+    ```bash
+    Registered Providers: [
+    '0xa47C8AdfFc62FE17fC39e23267B774879b0ABE6b',
+    '0x73Ed2ff15e848589B752a64aBa0f3B8B4bEdd6DD',
+    '0xf956071E0e58cd9cb68fA0e4CC86aEc7C8bEd586',
+    '0xb0B7858789393CE5A1B739359310D784f400bFFE',
+    '0x380307325EFf7137dC96431F45599EBfc4E2A6e9',
+    '0x2C4a6d656d9756E5D21082C5317ED93464B370Fb',
+    '0xc260620078801de0019877138f85eC0e30B9c643',
+    '0x6E25039D4B24630c40010934cFf94443eC939C8b',
+    '0x539077C1A1C0f489Fc6F23305750990341B664AF',
+    '0x2E6bC387a850397c0496Cd70355d7B87Ac9A76e0',
+    '0xB56700fc7f1Cc45dea6f575e7014bb2EE8CEEC1F',
+    '0x028677f17372b4B19C49Cfa74A92654d18eDfe49',
+    '0x051f91FeC4421bfDC5A4504E123ABb73C7790Ed1',
+    '0x762bba6B80dcFaf65f9F91A6a6ed60559f4F003c',
+    ...
+    ]
+    ```
+  </Accordion>
+
+  <Accordion title="Additional Information" icon="info">
+    Check out the [Provider Registry Explorer](http://explorer.testnet.mev-commit.xyz/address/0x5960774AD41D03DAB4916a30bD2190f8b3b3b4b2) for more details on registered providers.
```

Comment:
We should use the `ProviderRegistryExplorerUrl` var here right?

Reply:
I literally spent an hour trying to get that to work on mvintlify. Not wasting more time on it. Will go ahead and delete it for now. We can explore this kind of code organization later.